tagstudio/src/qt/ts_qt.py

```diff
@@ -186,9 +183,21 @@ def __init__(self, core, args):
 
         self.SIGTERM.connect(self.handleSIGTERM)
 
-        self.settings = QSettings(
-            QSettings.IniFormat, QSettings.UserScope, "tagstudio", "TagStudio"
-        )
+        if self.args.config_file:
+            path = Path(self.args.config_file)
+            if not path.exists():
+                logging.warning(
+                    f"[QT DRIVER] Config File does not exist creating {str(path)}"
+                )
+            logging.info(f"[QT DRIVER] Using Config File {str(path)}")
+            self.settings = QSettings(str(path), QSettings.IniFormat)
+        else:
+            self.settings = QSettings(
+                QSettings.IniFormat, QSettings.UserScope, "TagStudio", "TagStudio"
+            )
+            logging.info(
+                f"[QT DRIVER] Config File not specified, defaulting to {self.settings.fileName()}"
```

Comment:
This should probably have better wording, right now it sounds like the user did something wrong if they didn't specify a config file
```suggestion
                f"[QT DRIVER] Using local config stored in {self.settings.fileName()}"
```